docs/API.md

```diff
@@ -1671,23 +1671,24 @@ hooks in the plugins bundled with Flot.
     The resize hook is used to be notified after the plot was resized.
 
  - findNearbyItems   [phase 7]
-     ```function (canvasX, canvasY, seriesFilter, distance)```
+     ```function (canvasX, canvasY, series, radius, computeDistance, items)```
     
     The findNearbyItems hook is used to extend the default behavior of
-    the flot-provided 'findNearbyItems' function. The hover plugin is
-    an example of a plugin that leverages this method to return items 
-    that will be highlighted when hovered over. So, as an example, a new
-    plugin might provide this hook to allow the hover plugin to highlight
-    points that the flot-provided findNearbyItems method would have 
-    otherwise not found.
+    the flot-provided 'findNearbyItems' function. The hover plugin will take
+    advantage of this new capability by reporting the whole set of items
+    discovered by findNearbyItems (including the ones the user-defined hook
+    has found) in the args of the hover or click events it will raise. This
+    allows user to describe non-default behaviors for items found on hover or 
+    click through their hook.
     
-    canvasX and canvasY are the plot-space coordinates that are being
-    hovered over. Being in plot-space and not canvas-space, these may
-    be negative as the plot generally does not take up the whole 
-    canvas.
+    canvasX and canvasY are the canvas-space coordinates used to search from 
+    for the nearby items.
     
-    seriesFilter is a function to determine if the series should be
-    checked.
+    series is the data series that was allowed by the seriesFilter function
```

Comment:
I don't think this is super clear. How about: 

"series is the current data series for which the `findNearbyItems` hook should return nearby data. The hook will be called for each data series separately, as long as it is not filtered out by the `seriesFilter` function used by the caller."

Reply:
Much better. Words hard.